tests/integration/kubernetes/gha-run.sh

```diff
@@ -98,6 +98,20 @@ EOF
 
 	sleep 60s
 	sudo cat "${containerd_config_file}"
+
+	sudo systemctl restart k3s
```

Comment:
I think this should be placed in the `k3s` arm inside the `case` because we cannot guarantee if `k3s` is used outside the `case`.